### ISSUE TITLE
Local `compile-proto.sh` Script Pollutes `go.mod`

### ISSUE THREAD

**Issue by coltmcnealy-lh** [littlehorse-enterprises/littlehorse#808]
The `compile-proto.sh` script runs a Docker container with a volume mounted at the base of this repo. It also runs some `go get` and `go install` commands inside the container. The problem is that these commands end up polluting the `go.mod` in the base of our repo.

When I run `go mod tidy` on the `master` branch, I get the following diff, which is Not Good:

```
index ad13a6d6..7e81e453 100644
--- a/go.mod
+++ b/go.mod
@@ -14,28 +14,14 @@ require (
        google.golang.org/protobuf v1.31.0
 )
 
-require (
-       github.com/Masterminds/semver v1.4.2 // indirect
-       github.com/Masterminds/sprig v2.15.0+incompatible // indirect
-       github.com/aokoli/goutils v1.0.1 // indirect
-       github.com/envoyproxy/protoc-gen-validate v0.10.1 // indirect
-       github.com/gogo/protobuf v1.1.1 // indirect
-       github.com/huandu/xstrings v1.0.0 // indirect
-       github.com/imdario/mergo v0.3.4 // indirect
-       github.com/mwitkow/go-proto-validators v0.0.0-20180403085117-0950a7990007 // indirect
-       github.com/pseudomuto/protoc-gen-doc v1.5.1 // indirect
-       github.com/pseudomuto/protokit v0.2.0 // indirect
-       golang.org/x/net v0.12.0 // indirect
-       google.golang.org/genproto v0.0.0-20230530153820-e85fd2cbaebc // indirect
-       google.golang.org/genproto/googleapis/api v0.0.0-20230530153820-e85fd2cbaebc // indirect
-)
+require golang.org/x/net v0.12.0 // indirect
```

**Comment by coltmcnealy-lh** [comment]
Was this fixed in the `0.11.2` release process @mijailr and @eduwercamacaro ?

**Comment by coltmcnealy-lh** [comment]
Appears to have been fixed.
